src/processing.rs

```diff
@@ -213,13 +220,16 @@ pub fn derivative_extraction(feat: Array2<f32>, DeltaWindows: i32) -> Array2<f32
         let dif = Range * FEAT.slice(s![.., offset + Range..offset + Range + cols])
             - FEAT.slice(s![.., offset - Range..offset - Range + cols]);
 
-        Scale += 2 * np.power(Range, 2);
+        Scale += 2 * Range ^ 2;
```

Comment:
In Rust (and in C/C++) "^" is actually XOR, to do the square operation wolud be Range.pow(2)

Reply:
ah, my bad. I'll fix that